components/prism-bro.js

```diff
@@ -0,0 +1,102 @@
+Prism.languages.bro = {
+    'comment': [
+        {
+	        pattern: /(^|[^\\$])#.*/,
+		            	lookbehind: true
+		            }
+	],
+
+    'string': /(["'])(\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+
+    'boolean': /\b(T|F)\b/,
+
+    'keyword': [
+        /\b(break|next|continue)\b/,
+        /\b(alarm|using|of|add|delete)\b/,
+		/\b(export)\b/,
+        /\b(print|return|schedule)\b/,
+		/\b(when|timeout)\b/,
+		/\b(addr|any|bool|count)\b/,
+		/\b(double|enum)\b/,
+		/\b(file|int|interval)\b/,
+		/\b(pattern|opaque)\b/,
+		/\b(port|record|set)\b/,
+		/\b(string|subnet|table)\b/,
+		/\b(time|vector)\b/,
+		/\b(for|if|else)\b/,      
+	],
+	
+	'operator': /--?|\+\+?|!=?=?|<=?|>=?|==?=?|&&|\|\|?|\?|\*|\/|~|\^|%/,
+	
+	'function': [
```

Comment:
You can combine that into:

```js
'function': {
    pattern: /(?:function|hook|event) [a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)?/,
    inside: {
        keyword: /^(?:function|hook|event)/
    }
},
```

You should limit the inside keyword to the start of the line by using `^`.